app/ui/browser/views/page/index.jsx

```diff
@@ -175,6 +175,12 @@ function addListenersToWebView(webview, pageAccessor, dispatch) {
       return;
     }
 
+    // If a page is aborted (like hitting BACK/RELOAD while a page is loading), we'll get a
+    // load failure of ERR_ABORTED with code `-3` -- this is intended, so just ignore it.
+    if (Math.abs(errorCode) === 3) {
```

Comment:
They come in as `-3` -- should maybe save it from here, or add utilities to abstract this difference out, but wondering where else it'll play out, but yeah, in the future, I definitely see some helper functions here.